backend/django/test/api/test_api_annotate.py

```diff
@@ -505,17 +500,17 @@ def test_unlabeled_table(
         {"labelID": test_labels[0].pk, "labeling_time": 1},
     )
     response = admin_client.get(
-        "/api/data_unlabeled_table/" + str(project.pk) + "/"
+        "/api/data_unlabeled_table/?project=" + str(project.pk)
```

Comment:
I'd prefer to start migrating to using f-strings.  It might be odd at the start since all of the old code doesn't use them, but if you make a change that can use f-strings then go ahead

Reply:
So... I realize you were not intending me to replace all the string concats with f-strings.... but once I started doing it I couldn't figure out a good stopping point. So yeah. Most of them are now f-strings.

Reply:
No complaints from me.  If we've already gone this far then might as well go all the way.  There are a few more places that could be replaced.  Use the follow search-all regex

`".*" \+`

Reply:
I think I got all of them on the backend now.